string mode
float64 joint1
float64 joint2
float64 joint3
float64 t
---
string message
